Tasks/Rybina/Task1/Task1.py

```diff
@@ -0,0 +1,17 @@
+#variant1
+
+deposit = 20000
+period = 60
+rate = 15/12/100
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice